guides/source/active_record_querying.md

```diff
@@ -1266,6 +1266,9 @@ class Client < ActiveRecord::Base
 end
 ```
 
+NOTE: The `default_scope` is also applied while creating/building a record.
+It is not applied while updating a record.
```

Comment:
@repinel thanks for your feedback. I took a look in some notes in random sections of the guides and none of shows a example of code, I think the Idea is to be a small note only.